docker-compose.yml

```diff
@@ -33,56 +33,13 @@ services:
         max-file: "5"
     restart: "no"
     container_name: zkwasm-rocksdb-data
-  prover-dry-run-service:
+  prover-node:
```

Comment:
We removed the prover-dry-run-service. So I remember we need check all stop/upgrade/start scripts. I remember in those scripts we have some mechanism to show logs in terminals and need refine.